dependencyManagement/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val TEST_SNAPSHOTS = rootProject.findProperty("testUpstreamSnapshots") == "true"
 
 // This is the version of the upstream instrumentation BOM
-val otelVersion = "1.32.1"
+val otelVersion = "1.32.1-adot1"
```

Comment:
Sorry, why do we need this change?

Reply:
We need to update the otel dependency to use version `1.32.1-adot1`, so that it can use our patch agent instead of upstream default agent. 
We also override the version in patch file, it can be found at the end of .github/patches/opentelemetry-java-instrumentation.patch:
```
--- a/version.gradle.kts
+++ b/version.gradle.kts
@@ -1,5 +1,5 @@
-val stableVersion = "1.32.1"
-val alphaVersion = "1.32.1-alpha"
+val stableVersion = "1.32.1-adot1"
+val alphaVersion = "1.32.1-adot1-alpha"
```